lib/actions.ex

```diff
@@ -0,0 +1,46 @@
+defmodule Actions do
+  def subtract(attr1, attr2)
+      when is_integer(attr1) or (is_float(attr1) and is_integer(attr2)) or is_float(attr2) do
```

Comment:
А как вы предполагаете складывать целое со списком? Потому что в этот гард пролезет `attr2 = [1, 2, 3]`

```suggestion
      when is_number(attr1) and is_number(attr2)
```